docs/examples/connector_x_arrow/connector_x_arrow.py

```diff
@@ -0,0 +1,72 @@
+"""
+---
+title: Load mysql table with ConnectorX & Arrow
+description: Load data from sql queries fast with connector x and arrow tables
+keywords: [connector x, pyarrow, zero copy]
+---
+
+The example script below takes genome data from public **mysql** instance and then loads it into **duckdb**. Mind that your destination
+must support loading of parquet files as this is the format that `dlt` uses to save arrow tables. [Connector X](https://github.com/sfu-db/connector-x) allows to
+get data from several popular databases and creates in memory Arrow table which `dlt` then saves to load package and loads to the destination.
+:::tip
+You can yield several tables if your data is large and you need to partition your load.
+:::
+
+We'll learn:
+
+- How to get arrow tables from [connector X](https://github.com/sfu-db/connector-x) and yield them.
+- That merge and incremental loads work with arrow tables.
+- How to enable [incremental loading](../general-usage/incremental-loading) for efficient data extraction.
+- How to use build in ConnectionString credentials.
```

Comment:
so we don't have Setup: Running this example on your machine section anymore?
do you think it's useless or it difficult to add in this implementation?

Reply:
it was a feedback from @burnash (at least if i recall correctly) to make these pages more concise. I can add this section back in though, i am not super opinionated on this.